.github/workflows/windows.yml

```diff
@@ -15,23 +15,27 @@ jobs:
     runs-on: windows-latest
 
     strategy:
+      fail-fast: false
       matrix:
-        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
FYI, Neon follows LTS and will support Node 10 until it goes out of maintenance 4/30.

I support removing it from the build matrix, but we may want to defer until after or closer to the date.

Reply:
Good point. Maybe we could go from a full factorial matrix to a more sparse configuration. Not every node version would have to be tested on every Rust toolchain and every target.

Reply:
I would like to do that for the ARM builds. We can do a smaller matrix on PRs and the full matrix on merge to main.